modules/ROOT/pages/coverage-maven-concept.adoc

```diff
@@ -3,13 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-partial:
+:page-deployment-options: acb-ide, studio-ide
 
 // tag::coverage-report[]
 
-Since MUnit 2.x, the coverage report goal is integrated with the maven reporting section. Coverage Reports are generated during Maven's `site` lifecycle, during the `coverage-report` goal.
+Since MUnit 2.x, the coverage report goal is integrated with the Maven reporting section. MUnit generates Coverage Reports during the Maven `site` lifecycle, during the `coverage-report` goal.
 
-The following configurations apply only when you execute your MUnit tests using the Maven plugin. They do not apply when running tests from Studio. +
-To run coverage reports in Studio see xref:coverage-studio-concept.adoc[Using Coverage in Studio].
+The following configurations apply if you execute your MUnit tests using the Maven plugin. They don't apply when running tests from Anypoint Code Builder or Anypoint Studio.
```

Comment:
Is this going to be easy to maintain:

"They don't apply when running tests from Anypoint Code Builder or Anypoint Studio."